src/platform/site-wide/cta-widget/CallToActionAlert.jsx

```diff
@@ -18,10 +20,19 @@ export default function CallToActionAlert({
       <div>
         {alertText}
         {buttonText && (
-          <button className={buttonClass} onClick={buttonHandler}>
+          <button
+            className={buttonClass}
+            onClick={buttonHandler}
+            style={secondaryLinkText ? { marginRight: '1em' } : {}}
+          >
             {buttonText}
           </button>
         )}
+        {secondaryLinkText && (
+          <a href="#" onClick={secondaryLinkHandler}>
```

Comment:
Is there a better option than `href="#"`?

Reply:
Yes, this should be a `button` as well. Then you won't need to set the styles attribute. If you need it to be look like a link, use the `va-button-link` class.